### PR TITLE
Feat/UI competencies

### PR DESCRIPTION
**Pull request by jona159**

### CODE REVIEW

**Comment by jona159** [comment]
Some redundant code, maybe need to think about how to improve it

File: src/app/components/accordion.component.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+			<button
+				class="tw-w-40 tw-px-2 tw-py-2 tw-bg-white tw-rounded-xl tw-border tw-border-solid tw-border-[var(--color-purple)]"
+				hlmAccordionTrigger
+			>
+				{{ 'General.showDetails' | translate }}
+				<hlm-icon hlmAccIcon />
+			</button>
+			<brn-accordion-content hlm>
+				<div class="tw-flex tw-gap-4 tw-mt-2">
+					<span class="tw-font-semibold">Kategorie: </span>
```
Comment (YouQam) [inline src/app/components/accordion.component.ts:33]: add a translation for `Kategorie`

File: src/app/public/components/badge-assertion/badge-assertion.component.html [hunk 2]
```diff
@@ -190,6 +183,46 @@ <h2 class="u-text-h3-semibold u-margin-bottom2x u-hidden-mintablet">Description<
...
+					<h3 class="u-text-h3-semibold u-text-dark2 u-margin-bottom2x u-margin-top4x">Kategorie:</h3>
+					<span>{{ categoryOptions[badgeClass['extensions:CategoryExtension'].Category] }}</span>
+				</section>
+				<!-- Kompetenzen -->
+				<section>
+					<h3
+						class="u-text-h3-semibold u-text-dark2 u-margin-bottom2x u-margin-top4x"
+						*ngIf="badgeClass['extensions:CompetencyExtension'].length > 0"
+					>
+						Kompetenzen
```
Comment (YouQam) [inline src/app/public/components/badge-assertion/badge-assertion.component.html:197]: Add a translation here also

**Review by YouQam — CHANGES_REQUESTED** [review]
LGTM
I just commented on some words that needs translation. Then PR is ready to be merged.

**Review by timber-they — APPROVED** [review]

**Review by YouQam — APPROVED** [review]

**Pull request merged**